version: "3.7"

services:
   api:
      build:
         context: .
         dockerfile: Dockerfile
      command: python manage.py runserver 0.0.0.0:8000
      volumes:
         - .:/app
      ports:
         - 8000:8000
      env_file:
         - .env
      restart: "on-failure"
      depends_on:
         - db
      networks:
         - bridge

   db:
      image: geodjango-api
      ports:
         - 5432:5432
      volumes:
         - postgres_data:/var/lib/postgresql
      environment:
         - POSTGRES_USER=romanradionov
         - POSTGRES_PASS=89250427004
         - POSTGRES_DBNAME=geodjango_rn
      restart: "on-failure"
      networks:
         - bridge



networks:
   bridge:
      driver: bridge

volumes:
   postgres_data: